pre-commit

```diff
@@ -10,6 +10,8 @@ if [ -L "$0" ]; then
 	else
 		DEV_LIB_PATH=$( dirname $( readlink "$0" ) )
 	fi
+elif [ -e "$(dirname "$0")/.dev-lib-path" ]; then
+    DEV_LIB_PATH="$(realpath $(cat "$(dirname "$0")/.dev-lib-path"))"
 else
```

Comment:
It works on Windows, and that's the handicapped environment 😂 , exists on OS X and Linux, can't think of others. Here is a possible reference for availability : http://www.math.utah.edu/~beebe/unix/r/realpath.html